README.md

```diff
@@ -38,6 +38,18 @@ to your Github account:
 You should now receive an email approximately every two weeks containing the changes made to the APIs since the last update.
 
 * Please note that the date shown is the date that the change was deployed to the Sandbox or Production.
+---
+### XX August 2023 (TBC)
+
+The following change was deployed into sandbox:
+
+#### self-assessment-bsas-api
+
+New API Version v4.0 is now available
+
+* For `v4.0`:
+  * Updated data fields within the `adjustableSummaryCalculation` object to accept both positive and negative values for the`Retrieve Self-Employment BSAS` endpoint.
```

Comment:
Needs to mention the endpoint(s) affected

Reply:
Actually just noticed it does, but ought to be at the start of the sentence!
Ideally this:

`Retrieve Self-Employment BSAS` endpoint: data fields within the `adjustableSummaryCalculation` response object can now return negative values.